src/components/AnswersOptions/tests/AnswersOptions.test.tsx

```diff
@@ -0,0 +1,16 @@
+import {fireEvent, render, screen} from '@testing-library/react-native';
+import {AnswersOptions} from '@src/components/AnswersOptions/index';
+
+const onPressMock = jest.fn();
+const AnswersOptionsProps = {
+  onPress: onPressMock,
+  data: ['one', 'two', 'thee'],
+};
+
+test('Should render view block, correct operation of btn', async () => {
```

Comment:
btn

Reply:
also try to describe test as concrete as possible. What does mean "correct operation"? I don't know condition of correctness

Reply:
and add tests for styles